src/pages/Kvittering.less

```diff
@@ -0,0 +1,8 @@
+.typo-innholdstittel {
+  margin-bottom: 20px;
+}
+
+.alertstripe {
```

Comment:
Not good. Dette vil endre på alle alertstriper.

Reply:
Bare inne i denne som har importert less-filen, da? Eller?

Reply:
Nope, i hele DOM'en. Hvis du vil endre på styling bør du have en parent div og bruke det med som selector. Eg have en `<div className="kvittering"></div>` rundt og bruke `.kvittering { .alertstripe { etc`

Reply:
Yess, det gjorde jeg! :D

Reply:
Bra du passer på!

Reply:
:D